src/ansible_navigator/cli.py

```diff
@@ -75,6 +77,37 @@ def pull_image(args):
         error_and_exit_early(image_puller.assessment.exit_messages)
 
 
+class Formatter(logging.Formatter):
```

Comment:
Since the class is not specific to navigator cli functionality this can be moved to `utils/formatter.py` along with some of the other formatting function like `now_iso`?

Reply:
I was thinking about a follow-up PR that moved both the Formatter and setup_logger function into its own file called logging. I didn't want to do it in this PR because the diff for setup_logger would be lost with the mew file.  I do think they should stay together tho